package.json

```diff
@@ -3,18 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.3.1",
```

Comment:
why did you use bootstrap for this project?

Reply:
To use button component.

Reply:
Well, can you use div to replicate that button?
Because if we use bootstrap, some of our old components will not look properly...